packages/shop/src/containers/search-result/SearchResult.tsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import 'styled-components/macro';
+import { useTranslation } from 'react-i18next';
+import { useRouter } from 'next/router';
+
+import { ErrorDisplay } from '@rmp-demo-store/ui/error-display';
+import Spinner from '@rmp-demo-store/ui/spinner';
+import Stack from '@rmp-demo-store/ui/stack';
+
+import useSearch from '../../hooks/use-search';
+import SingleProduct from '../../components/product/single';
+import { ProductDisplayItem } from '../../components/product/types';
+import { space } from '@rmp-demo-store/ui/theme-utils';
```

Comment:
nit: sort import order

Reply:
Fixed in https://github.com/moloco-rmp/demo-store/pull/10/commits/c2cd71bfd2ef4561a4c2c825a3064eb766b6ff28.